connectors-and-apis.md

```diff
@@ -1,92 +0,0 @@
----
-title: Connectors and APIs
```

Comment:
This file has been there for years. Some of its contents may be outdated. We don't want users to see the entry so we remove it from TOC but still keep the file.